src/components/KitchenModal/KitchenModal.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Modal from "../Modal";
+import UserTableRow from "../UserTableRow";
+import {
+  DialogTitle,
+  DialogContent,
+  TextField,
+  DialogActions,
+  Table,
+  TableHead,
+  Typography,
+  Button,
+} from "@mui/material";
+
+const KitchenModal = ({
+  newKitchenUsers,
+  setNewKitchenUsers,
+  addNewUser,
+  closeModal,
+  open,
+  KitchenName,
+  setKitchenName,
+  saveKitchen,
+}) => {
```

Comment:
Since KitchenModal is going to be used in multiple places we should make the props list as small as possible. It makes sense to keep open/close as props since that will change based on where it's used (for example, open/close is set by a menu item in the appBar, but in the kitchens page its a button listed by the other kitchens). But kitchenUsers, addNewUser, KitchenName, and saveKitchen should all be inside of this modal!

```suggestion
const KitchenModal = ({
  closeModal,
  open,
}) => {
```

Reply:
(also, while we're here, maybe let's keep the name open/close consistent! Either "open" and "close" or "openModal" and "closeModal")